Update min. required autowrap in pyOpenMS (new operators)

Sorry if this is already an open issue but it s been a few weeks that I m not able to build pyopenms. 
```
Consolidate compiler generated dependencies of target pyopenms_7
[ 92%] Built target Tutorial_AASequence
[ 92%] Built target Tutorial_MapAlignment
[ 92%] Building CXX object src/pyOpenMS/CMakeFiles/pyopenms_7.dir/__/__/pyOpenMS/pyopenms/pyopenms_7.cpp.o
/Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/pyopenms_7.cpp:1:2: error: Do not use this file, it is the result of a failed Cython compilation.
#error Do not use this file, it is the result of a failed Cython compilation.
 ^
1 error generated.
make[2]: *** [src/pyOpenMS/CMakeFiles/pyopenms_7.dir/__/__/pyOpenMS/pyopenms/pyopenms_7.cpp.o] Error 1
make[1]: *** [src/pyOpenMS/CMakeFiles/pyopenms_7.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
Consolidate compiler generated dependencies of target pyopenms_8
Consolidate compiler generated dependencies of target pyopenms_6
Consolidate compiler generated dependencies of target pyopenms_2
[ 92%] Linking CXX shared module ../../pyOpenMS/pyopenms/pyopenms_8.cpython-39-darwin.so
[ 92%] Linking CXX shared module ../../pyOpenMS/pyopenms/pyopenms_6.cpython-39-darwin.so
[ 92%] Linking CXX shared module ../../pyOpenMS/pyopenms/pyopenms_2.cpython-39-darwin.so
Fixing typing information for pyopenms_8
Fixing typing information for pyopenms_6
Fixing typing information for pyopenms_2
Traceback (most recent call last):
  File "/Users/eeko/openms-develop/OpenMS/src/pyOpenMS/fix_pyi_imports.py", line 6, in <module>
Traceback (most recent call last):
  File "/Users/eeko/openms-develop/OpenMS/src/pyOpenMS/fix_pyi_imports.py", line 6, in <module>
    with open(args[0], 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/pyopenms_6.pyi'
    with open(args[0], 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/pyopenms_8.pyi'
make[2]: make[2]: *** [pyOpenMS/pyopenms/pyopenms_8.cpython-39-darwin.so] Error 1*** [pyOpenMS/pyopenms/pyopenms_6.cpython-39-darwin.so] Error 1

make[2]: make[2]: *** Deleting file `pyOpenMS/pyopenms/pyopenms_6.cpython-39-darwin.so'
*** Deleting file `pyOpenMS/pyopenms/pyopenms_8.cpython-39-darwin.so'
make[1]: *** [src/pyOpenMS/CMakeFiles/pyopenms_6.dir/all] Error 2
make[1]: *** [src/pyOpenMS/CMakeFiles/pyopenms_8.dir/all] Error 2
Traceback (most recent call last):
  File "/Users/eeko/openms-develop/OpenMS/src/pyOpenMS/fix_pyi_imports.py", line 6, in <module>
    with open(args[0], 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/eeko/openms-develop/openms_build/pyOpenMS/pyopenms/pyopenms_2.pyi'
make[2]: *** [pyOpenMS/pyopenms/pyopenms_2.cpython-39-darwin.so] Error 1
make[2]: *** Deleting file `pyOpenMS/pyopenms/pyopenms_2.cpython-39-darwin.so'
make[1]: *** [src/pyOpenMS/CMakeFiles/pyopenms_2.dir/all] Error 2
make: *** [all] Error 2

```

--- a/src/pyOpenMS/CMakeLists.txt
+++ b/src/pyOpenMS/CMakeLists.txt
@@ -167,7 +167,7 @@ if(AUTOWRAP_MISSING)
 else()
     execute_process(
         COMMAND
-        ${Python_EXECUTABLE} -c "import autowrap; exit(autowrap.version >= (0, 22, 5))"
+        ${Python_EXECUTABLE} -c "import autowrap; exit(autowrap.version >= (0, 22, 8))"
         RESULT_VARIABLE _AUTOWRAP_VERSION_OK
         ERROR_QUIET
         OUTPUT_QUIET
@@ -188,7 +188,7 @@ else()
         message(STATUS "Looking for autowrap - found autowrap ${AUTOWRAP_VERSION}, version ok")
         set(AUTOWRAP_VERSION_OK TRUE)
     else()
-        message(STATUS "Found autowrap version ${AUTOWRAP_VERSION}. The version is too old (>= 0.22.5 is required)")
+        message(STATUS "Found autowrap version ${AUTOWRAP_VERSION}. The version is too old (>= 0.22.8 is required)")
         message(FATAL_ERROR "Please upgrade autowrap or disable pyOpenMS.")
     endif()
 endif()

